packages/core/config/webpack.common.js

```diff
@@ -197,7 +199,7 @@ module.exports = (webpackEnv) => {
     plugins: [
       !isEnvProduction && new webpack.HotModuleReplacementPlugin(),
       new HtmlWebpackPlugin({
-        title: path.basename(redwoodPaths.base),
+        title: redwoodConfig.web.title || path.basename(redwoodPaths.base),
```

Comment:
I'd recommend the opposite:
` page title || site title`

When looking through browser tabs, the favicon will tell you site. The page title is what you want to see ('cause shortened snippet), otherwise you get the site name on repeat in tabs.

Reply:
Yes that make sense, and when I compares many website is like 50/50 I don't understand why. But at the end I think we go with a custom template so is up to you to decide the format. Also here is a failover for the site title only.

Reply:
Also I don't think we need this anymore if we have default title inside RedwoodProvider